Makefile

```diff
@@ -49,6 +49,9 @@ pb: pb_rawapi pb_ibft pb_synccommittee
 
 SOL_FILES := $(wildcard nil/contracts/solidity/tests/*.sol nil/contracts/solidity/*.sol)
 BIN_FILES := $(patsubst nil/contracts/solidity/%.sol, contracts/compiled/%.bin, $(SOL_FILES))
+CHECK_LOCKS_DIRECTORIES := ./nil/internal/network
```

Comment:
You can implement `make checklocks` and use it inside lint and workflow

Reply:
Added the `checklocks` target. I did not add it to `lint` yet. It doesn't work very fast on the whole project, perhaps it will be an unpleasant surprise for those who are used to using `make lint`.